single-editorial.php

```diff
@@ -28,77 +28,88 @@
     if (!empty($slides)) {
       if ($is_recent) {
 ?>
-
+      
       <!-- Product details container -->
-      <div class="col col-s-12 col-m-3 set-content-height">
+      <div class="col col-s-10 offset-s-1 col-m-3 offset-m-0 column justify-between">
```

Comment:
what is the purpose of `column justify-between`? It doesn't seem to be doing anything that I can see on my local